test/components/kubernetes/cert_rotate_disruptive_test.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2021 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build packet baremetal aws
+// +build disruptivee2e
+
+package kubernetes_test
+
+import (
+	"bytes"
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/kinvolk/lokomotive/cli/cmd/cluster"
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+)
+
+func TestCertificateRotate(t *testing.T) { //nolint:funlen
```

Comment:
```suggestion
func Test_Certificate_rotation(t *testing.T) { //nolint:funlen
```

Reply:
Why again?

Reply:
Ah right, we agreed not to do it for now. Well, it's just a suggestion for better readability :)